Code/Framework/AzCore/AzCore/Memory/Memory_fwd.h

```diff
@@ -166,7 +166,7 @@
     /* ========== standard operator new/delete ========== */                                                                                                                        \
     AZ_FORCE_INLINE void* operator new(std::size_t size) {                      /* default operator new (called with "new _Class()") */                                             \
         AZ_Assert(size == sizeof(_Class), "Size mismatch! Did you forget to declare the macro in derived class? Size: %d sizeof(%s): %d", size, #_Class, sizeof(_Class));           \
-        AZ_Warning(0, true/*false*/, "Make sure you use aznew, offers better tracking! (%s)", #_Class /*Warning temporarily disabled until engine is using AZ allocators.*/);       \
+        AZ_Warning(AZ::Debug::Trace::GetDefaultSystemWindow(), true/*false*/, "Make sure you use aznew, offers better tracking! (%s)", #_Class /*Warning temporarily disabled until engine is using AZ allocators.*/);       \
```

Comment:
The disabled warning doesn't seem very temporary. :(

Reply:
And that is why TODOs left in code are useless.
A triaged issue that exist in a GHI to revisit the problem would be better.
However this warning is completely useless to keep because, `aznew` is just a defined to `new` without any parameters, so there is no benefit at all to using [aznew](https://github.com/o3de/o3de/blob/development/Code/Framework/AzCore/AzCore/Memory/Memory_fwd.h#L19) in the first place.

I'll remove this line